shell/mixins/chart.js

```diff
@@ -185,6 +186,10 @@ export default {
         }
       }
 
+      if (this.chart?.id === OPA_GATE_KEEPER_ID) {
+        warnings.unshift(this.t('gatekeeperIndex.deprecated', {}, true));
+      }
```

Comment:
Is this the recommended way to show a deprecation warning? I'm noticing that there's already a `showDeprecated` computed prop and I'm wondering if that's used for the same thing.

Reply:
`showDeprecated` is not used for showing deprecation messages. We filter the charts that have a `deprecated` property set to `true`, but if `showDeprecated` is also `true` then we bypass this filter.